keycloak.yaml

```diff
@@ -38,6 +38,9 @@ pipeline:
       export LANG=en_US.UTF-8
       export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
 
+      # Update xmlsec from 2.2.3 to 2.2.6 to fix GHSA-xfrj-6vvc-3xm2
+      sed -i 's/2.2.3/2.2.6/g' pom.xml
```

Comment:
This seems a little brittle to me, idk — what do you think about creating a patch to achieve this? That seems to be the most common way we've been handling bumps in Java projects, including in `pom.xml` files